test/dispatchers/touchDispatcherTests.ts

```diff
@@ -52,6 +52,7 @@ describe("Dispatchers", () => {
         div = TestMethods.generateDiv(DIV_WIDTH, DIV_HEIGHT);
         const component = new Plottable.Component();
         component.anchor(div);
+        component.computeLayout();
```

Comment:
how were the tests failing that required you to add this? will users of plottable be required to add this to get their touch/mouse events to work?